packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -1394,6 +1394,9 @@ abstract class FlutterCommand extends Command<void> {
     return help;
   }
 
+  @override
+  void printUsage() => globals.printStatus(usage);
```

Comment:
Currently `FlutterCommand` just uses the `printUsage` function from the `Command` class of the [args](https://github.com/dart-lang/args) package, which in turn uses `print` to display usage:

https://github.com/dart-lang/args/blob/8d82f3e2b38759279421455682b6641ce4d75043/lib/command_runner.dart#L87-L91

Testing whether the usage information is being printed or not requires the usage to be instead displayed through `globals.printStatus`.